src/PHPStan/Collector/ContextFromControllerRenderMethodCallCollector.php

```diff
@@ -54,22 +48,27 @@ public function processNode(Node $node, Scope $scope): ?array
             return null;
         }
 
-        if (count($node->args) !== 2) {
+        if (count($node->args) < 2) {
```

Comment:
3 args is allowed.

I also need to allow 1 args cause
```
$this->render($foo)
```
is valid, and we need to report someone who forgot passing the context for a template.